client/resultsview/main.js

```diff
@@ -44,6 +44,13 @@ class Main {  // this is constructed at the bottom
         // the location of the script should be inside the body
         // so we don't need document.ready()
         this.$body = $('body');
+
+        $(document).ready(() => {
+
+            $(document).mousedown(this, (event) => this._mouseDown(event));
+            $(document).mouseup(this, (event) => this._mouseUp(event));
+            $(document).mousemove(this, (event) => this._mouseMove(event));
```

Comment:
do you need to wait for $(document).ready() here?

Reply:
I wasn't sure if document needed to be ready for the iframe comms to work.